packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -76,6 +75,8 @@ const AppModals = ({
     const url_params = new URLSearchParams(useLocation().search);
     const url_action_param = url_params.get('action');
 
+    const is_eu_user = [ContentFlag.LOW_RISK_CR_EU, ContentFlag.EU_REAL, ContentFlag.EU_DEMO].includes(content_flag);
+
```

Comment:
@mahdiyeh-deriv we have a flag show_eu_related_content in traders hub. Later we can use that flag directly.LGTM